src/dbnode/persist/fs/retriever_options.go

```diff
@@ -65,6 +67,7 @@ func NewBlockRetrieverOptions() BlockRetrieverOptions {
 		bytesPool:        bytesPool,
 		fetchConcurrency: defaultFetchConcurrency,
 		identifierPool:   ident.NewPool(bytesPool, ident.PoolOptions{}),
+		queryStats:       stats.NoOpQueryStats(),
```

Comment:
nit: `NoopQueryStats` I think is more conventional to the rest of the codebase (slightly... just did a check and we have both but Noop wins out slightly but the fact that we have both is a little annoying)